recipes/statsmodels/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "statsmodels" %}
+{% set version = "0.6.1" %}
+{% set build = 0 %}
+{% set md5 = "f7580ebf7d2a2c9b87abfad190dcb9a3" %}
+
+package:
+    name: {{ name }}
+    version: {{ version }}
+
+source:
+    fn: {{ name }}-{{ version }}.tar.gz
+    url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    md5: {{ md5 }}
+
+build:
+    number: {{ build }}
+    script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+    build:
+        - python
+        - setuptools
+        - cython
+        - numpy x.x
+        - scipy
+        - pandas
+        - patsy
+    run:
+        - python
+        - numpy x.x
+        - scipy
+        - pandas
+        - patsy
+test:
+    imports:
+        - statsmodels
+
+about:
+    home: https://www.statsmodels.org
+    license: BSD 3
```

Comment:
Yeah, @ocefpaf also tried his hand at this in PR ( https://github.com/conda-forge/conda-smithy/pull/93 ). However, it turns out there are many different ideas of an acceptable license out there. Not to mention there are many things we have to accept that are non-standard licenses (e.g. HDF5). As a result, I think our best hope of solving this is to write a "spell checker for licenses". If you have thoughts on doing this, please feel free to contribute to the conversation. 😉